sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StatefulProcessorHandleSuite.scala

```diff
@@ -144,8 +147,16 @@ class StatefulProcessorHandleSuite extends StateVariableSuiteBase {
     test(s"registering timeouts with timeMode=$timeMode should succeed") {
       tryWithProviderResource(newStoreProviderWithStateVariable(true)) { provider =>
         val store = provider.getStore(0)
+        val timerCFName = if (timeMode == "ProcessingTime") {
```

Comment:
Why not extracting this to the private method providing two CF names for given timeMode? Looks like we are introducing non-trivial number of duplicated code.

Reply:
Maybe not a private method but utility object, as I'm seeing the same code in other suite as well.